holidayextras/plugin-transformer
# update plugin-transformer

#### What does this PR do? (please provide any background)

https://hxshortbreaks.atlassian.net/browse/WEB-10360
this updates plugin-transformer to use the latest dependencies and using the new version of transformer from 
https://github.com/holidayextras/transformer/pull/20
, and using standardjs for linting

#### What tests does this PR have?
none

#### How can this be tested?

pull https://github.com/holidayextras/the-works/pull/487 and https://github.com/holidayextras/the-wall/pull/206 doing a fresh `npm install` on each, startup themeBlueprint and make sure that you can get through the booking flow
 
#### Any tech debt?
nope

- I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

By approving a review you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

---
